src/_pytest/assertion/util.py

```diff
@@ -439,7 +442,10 @@ def _compare_eq_cls(
         explanation += ["Differing attributes:"]
         for field in diff:
             explanation += [
-                ("%s: %r != %r") % (field, getattr(left, field), getattr(right, field))
+                ("%s: %r != %r") % (field, getattr(left, field), getattr(right, field)),
+                "",
+                "Recursive from previous comparison",
```

Comment:
The sentence is good but I worry it might be a bit confusing for some users. Some other suggestions are:

> Zoom into last differing attribute

> Drill down into last differing attribute

I'm fine with whatever you decide.

Reply:
Changed it to "Drill down into differing attribute X:"